filer/tests/admin.py

```diff
@@ -2508,6 +2510,12 @@ def test_multi_files_perms_for_restricted_descendants(self):
         self.assertEqual(File.objects.filter(restricted=False).count(), 0)
         self.assertEqual(Folder.objects.filter(restricted=False).count(), 0)
 
+    def test_truncate_filename(self):
+        jpeg_file = StringIO.StringIO('      JFIF') # jpeg signature
+        jpeg_file.name = 'empty{}'.format(12345)
```

Comment:
Minor, I don't think we need to use .format here. Just 'empty12345' should do

Reply:
will do